apps/prairielearn/src/pages/instructorAssessmentQuestions/popoverBreakVariants.ejs

```diff
@@ -0,0 +1,17 @@
+<p>
+  This will mark all students' current variants of this question as broken,
+  meaning that <b>all unsubmitted attempts will be lost.</b>
```

Comment:
Is this unsubmitted or ungraded?

Reply:
Yeah, we should better distinguish between the two. "Ungraded" is more correct here.

Reply:
Changed in 7e0b69c and 66edad4.